src/CMakeLists.txt

```diff
@@ -154,7 +154,7 @@ if(SWIG_FOUND)
 			# Create 'avtranscoder-java' shared lib
 			swig_add_module(avtranscoder-java java ${AVTRANSCODER_BINDING_FILE})
 			if(WIN32)
-				set_target_properties(avtranscoder-java PROPERTIES OUTPUT_NAME "avtranscoder-java-${AVTRANSCODER_VERSION}")
+				set_target_properties(avtranscoder-java PROPERTIES OUTPUT_NAME "avtranscoder-java-${AVTRANSCODER_VERSION_MAJOR}-${AVTRANSCODER_VERSION_MINOR}")
```

Comment:
Arg it does not seems to be a good idea, since widely used Java tools, like Maven, expects Jar named __nameJar-fullVersion.jar__.